pyanaconda/ui/gui/spokes/storage.py

```diff
@@ -457,9 +466,8 @@ def _doExecute(self):
             StorageCheckHandler.errors = str(e).split("\n")
             hubQ.send_message(self.__class__.__name__, _("Failed to save storage configuration..."))
             self.data.bootloader.bootDrive = ""
-            self.data.ignoredisk.drives = []
```

Comment:
Hm, seems like a noop with original intent of resetting ignoredisk.ignoredisk. Interesting, but nothing that should be handled in scope of this PR I think.

Reply:
I can replace it with `self._disk_select_observer.proxy.SetIgnoredDisks([])` or just remove it. I don't want to keep the original line since UI shouldn't access the `ignoredisk` command anymore. However, I could at least add a comment here.

Reply:
I guess it is there for quite a long time, so let's just ignore it (ie I'm fine with the current patch).